requirements.txt

```diff
@@ -1,2 +1,4 @@
+git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```

Comment:
yeah, uploading to pypi would be ideal, but a git tag might be the easiest way to move forward for now 💭